src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -465,6 +465,7 @@ public void SetEncodingToNullThrows()
     }
 
     [Fact]
+    [Trait("Category", "SkipWhenLiveUnitTesting")]
```

Comment:
Are these tests skipped because thy apply GC pressure which we don't want to do inproc in LUT in VS?